znai-docs/znai/release-notes/1.53/_fix-2021-09-13-pydoc-assignment-parse-fix.md

```diff
@@ -1 +1 @@
-* Fix [variable extraction](snippets/python#content-full-content-variable) crashing in certain cases by adding more checks 
+* Fix [variable extraction](snippets/python#content-full-content-variable) crashing in certain cases by directly parsing the code rather than using the AST 
```

Comment:
I cut 53 release already, could you please move to 1.54? Sorry I forgot to update GitHub release

Reply:
Ah, didn't realise.  Updated.

Reply:
We need to move it to a separate `1.54` dir and do what you did to setup a new entry in release notes (I can do this part myself later).
I suggest we replace `fix` with the parsing of class variables - new feature

Reply:
That's exactly what I've done.

Reply:
Ah sorry! was looking at outdated commit :( thank you